src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -698,6 +700,17 @@ class LicenceService(
     )
   }
 
+  @Transactional
+  fun removeAPConditions() {
```

Comment:
Can we move this into it's own service in the `jobs` package like `LicenceActivationService`?

Reply:
created and moved in to RemoveApConditionsInPssPeriodService